app/helpers/tags_helper.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+module TagsHelper
+  # rubocop:disable MethodLength
```

Comment:
I think you can remove this disable comment. You can double check by running rubocop locally.